<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="IND-48NG453" failures="1" tests="3" name="FunctionalTest1" time="0.032" errors="0" timestamp="2021-03-14T21:34:44 IST">
  <testcase classname="TestCases.test1" name="LogOff" time="0.005">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestCases.test1.LogOff(test1.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- LogOff -->
  <testcase classname="TestCases.test1" name="Login" time="0.0"/>
  <testcase classname="TestCases.test1" name="ValidateUser" time="0.001"/>
</testsuite> <!-- FunctionalTest1 -->
